### ISSUE TITLE
Hardware issue - nonstandard camera connector pinout

### ISSUE THREAD

**Issue by fraopess** [mcauser/MCUDEV_DEVEBOX_H7XX_M#5]
Hi Mike! First of all thank you for you great work on this new board.  
This post is not for an issue with the code, but rather a warning about the hardware.
Micropyton connection works like a charm.

I tried to connect an OV2640 camera ([like this](https://fr.aliexpress.com/item/4000061425990.html?spm=a2g0o.search0301.0.0.f00556ebr9bAXS&algo_pvid=b8c4b341-18f0-4b07-bc1b-b4810095234d&algo_expid=b8c4b341-18f0-4b07-bc1b-b4810095234d-5&btsid=2100bb4a16085563192996615eb666&ws_ab_test=searchweb0_0,searchweb201602_,searchweb201603_)) to the 24pin connector on the back, but the camera started getting hot and finally burnt...
After reviewing the schematic I noticed that the camera connector pinout is not standard.
I tried to contact the vendor with little results.

Just be careful when you connect!
Any guess about what camera to connect?

**OV2640 flex cable pinout:**
![image](https://user-images.githubusercontent.com/5438387/102780324-00a68300-4396-11eb-91c8-94af16cd3fc1.png)

**MCUDEV camera pinout:**
![image](https://user-images.githubusercontent.com/5438387/102780395-23389c00-4396-11eb-9114-0b8130928af3.png)

**Comment by ljhtxdyma** [comment]
yes, I also found this issue...the camera is too hot to hurt me...I dont understand why they dont choose to use the standard DVP pinout...

**Comment by NoneType4Name** [comment]
I'll check it tomorrow, but u need to checkout pin's setup in CubeMx, bcs DCMI_D[0-1] difinitely doesn't matchs with default CubeMx pinouts for H750 DCMI. u need to setup this pins to other DCMI_D[0-1] pinouts.
